doc/advanced.adoc

```diff
@@ -124,6 +124,22 @@ Emitter<String> emitter;
 The default buffer size (`127`) can be configured using the `smallrye.messaging.emitter.default-buffer-size` property set using MicroProfile Config.
 Note that this value is only applied when `OnOverflow` is not used.
 
+=== Repeatable `@Incoming`
+
+You can use several `@Incoming on a single method:
+
+[source,java]
```

Comment:
Do we need to document `@Incomings` as well?

Reply:
Good question, I don't expect users to use it directly, but use multiple `@Incoming`. We should maybe say so in the javadoc. 

Unfortunately, it's how the repeatable annotations work, you create a container that would be "used" automatically.

Reply:
Ok, if it's purely for an implementation thing, then should `@Incomings` be in the api artifact or in the impl? And agree we need to doc it to indicate it's not for use